src/Apps/Artwork/Components/PricingContext.tsx

```diff
@@ -32,14 +32,19 @@ function PricingContext({ artwork }: PricingContextProps) {
       </Flex>
       <Spacer mb={[2, 3]} />
       <BarChart
-        minLabel={artwork.pricingContext.bins[0].minPrice}
+        minLabel={
+          artwork.pricingContext.bins[0].minPrice != null
```

Comment:
It looks like from vortex, we're returning a `0` correctly:
![image](https://user-images.githubusercontent.com/2081340/55904552-1eba5880-5b9e-11e9-8958-5ab2a7942614.png)

From metaphysics, we're returning a `null`:
![image](https://user-images.githubusercontent.com/2081340/55904571-2c6fde00-5b9e-11e9-8366-eadf36375f59.png)

So it seems in metaphysics we should be converting that `0` cents value into `$0` but aren't doing so. I remembered encountering this issue before: https://github.com/artsy/metaphysics/pull/1325 but I didn't end up revisiting like I said I would, and now it's biting us again!

Could you add a `TODO` here so we can revisit? I'll add an item to our sprint to revisit the metaphysics work, but that can be a separate work-item.

Reply:
Will do